Add ijam and tashkil to glossary (I18N-ACTION-1200) and fix addison's info

aphillips opened the pull request:
<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/aphillips/i18n-glossary/pull/15.html" title="Last updated on Sep 15, 2022, 5:11 PM UTC (b7db1a5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/i18n-glossary/15/231b8a7...aphillips:b7db1a5.html" title="Last updated on Sep 15, 2022, 5:11 PM UTC (b7db1a5)">Diff</a>

aphillips committed:
Fix addison's info

aphillips commented:
This didn't trigger the previewer tool and I don't seem to have write access to this repo. @xfq can you check it?

aphillips committed:
add ijam and tashkil

Sourced from https://r12a.github.io/scripts/arabic/arb.html#ijam_tashkil per TPAC discussion.

xfq commented:
Should be fixed now. You can push a commit to trigger the previewer tool, @aphillips.

aphillips committed:
added local.css

Our commonly used styles (like `uname`) were not present and we commented out the local.css file and didn't copy it in. I took the embedded styles and moved them to a copy of the specdev local.css file.

This does have presentational side effects. @r12a you should have a look at this. I could also just start building up a minimal `local.css` with just the local styles needed. Thoughts?

r12a commented on index.html:
```diff
@@ -248,6 +250,14 @@ <h2>Glossary</h2>
 
+<p><dfn id="def_ijam" class="lint-ignore export">Ijam</dfn>. An ijam is an Arabic script diacritical mark considered to be part of a basic letter form. &#x681; [<span class="uname" translate="no">U+0681 ARABIC LETTER HAH WITH HAMZA ABOVE</span>] is an example of a letter with an ijam. Unicode encodes letter+ijam combinations as separate, atomic characters, which are never given decompositions in  ...
+
+<aside class="note">
```
The glossary keeps definitions short on purpose, and limits the descriptions to one paragraph.  For this kind of additional information (some of which actually begins to wander off track) we should point out to somewhere else (probably your original source for this text, which is https://r12a.github.io/scripts/arabic/arb.html#ijam_tashkil).

r12a replied:
The definition really must have a link to the tashkil entry, and vice versa.

r12a replied:
The example of hah+hamza needs clarification to indicate that it represents a consonant in Pashto.  I will add that to the orthography description too.

r12a commented on index.html:
```diff
@@ -248,6 +250,14 @@ <h2>Glossary</h2>
+<p><dfn id="def_ijam" class="lint-ignore export">Ijam</dfn>. An ijam is an Arabic script diacritical mark considered to be part of a basic letter form. &#x681; [<span class="uname" translate="no">U+0681 ARABIC LETTER HAH WITH HAMZA ABOVE</span>] is an example of a letter with an ijam. Unicode encodes letter+ijam combinations as separate, atomic characters, which are never given decompositions in  ...
+
+<aside class="note">
+	<p>This distinction between using a character with <a>ijam</a> instead of combining a letter with a <a>tashkil</a> becomes important when choosing which Unicode characters to use because (as can be seen in the examples above) the visual forms can be identical. Using the wrong character can change the meaning of the text, affecting the results of text search, font rendering, text to speech, etc.< ...
```
I would like us to remove this paragraph and the following one.
Same applies to the tashkil section.

r12a commented on index.html:
```diff
@@ -39,7 +39,8 @@
           };
     </script>
-<!--link rel="stylesheet" href="local.css" /-->
+<link rel="stylesheet" href="local.css">
```
I intentionally commented out the local.css file.  We need to check what differences are made by including it.  If ok to include, we should reference the same file as other pages, so that we don't need to update in various locations.  If it needs tailoring, we should probably consider our options.

aphillips committed:
first pass at addressing @r12a's comments

r12a commented on index.html:
```diff
@@ -359,13 +353,7 @@ <h2>Glossary</h2>
-
-    <p>There are, however, some very common combinations of diacritic and base that can be represented using precomposed characters or decomposed sequences that are canonically equivalent. For those the standard encourages the use of the precomposed form, but the fact that the forms are canonically equivalent removes concerns about changes in meaning.</p>
-</aside>
+<p><dfn id="def_tashkil" class="lint-ignore export">Tashkil</dfn>. A tashkil (تَشْكِيل) is an Arabic script mark that indicates vocalization of text, as well as other types of phonetic guides to correct pronunciation. &#x62d;&#x654; [<span class="uname" translate="no">U+062D ARABIC LETTER HAH + U+0654 ARABIC HAMZA ABOVE</span>] is an example of a letter plus tashkil combination. Tashkil are separ ...
```
the arabic text needs markup and a lang tag

we should probably also add the arabic spelling for ijam if we have it here

aphillips committed:
changes from teleconference of 2022-09-15

Added correct codepoint markup.

Fixed internal termref styling.

Adjusted wording of the external link.

Converted Arabic term to NCRs.

@r12a Note that I tagged the tashkil example with language tag `ps` instead of `ar`. Is that appropriate?

aphillips committed:
added missing ampersand

aphillips commented:
Merging this per our discussion in 2022-09-15 teleconference. @r12a will make additional edits and also address the stylesheet issues. Note that this means that the ED is using the gigantor `local.css` file and we should probably wait to push this to TR until we resolve some of the CSS issues.
